.github/workflows/clippy_dev.yml

```diff
@@ -1,3 +1,5 @@
+# This action runs tests specific to `clippy_dev`, accessing through the `cargo dev` command. It
+# both checks the project as a whole and tests `clippy_dev`'s commands specifically.
```

Comment:
I think it would be great if we specified which steps are taking for checking the project as a whole.

```suggestion
# This action runs tests specific to `clippy_dev`, accessing through the `cargo dev` command. It
# both checks the project as a whole (formatting & updating the lints) and `clippy_dev`'s commands specifically.
```